internal/constants/constants.go

```diff
@@ -429,6 +429,9 @@ const ForumsURL = "https://community.activestate.com/c/state-tool/"
 // GlobalDefaultPrefname is the pref that holds the path to the globally defaulted project
 const GlobalDefaultPrefname = "projects.active.path"
 
+// LastUsedNamespacePrefname is the pref that holds the last used org for `state init`
```

Comment:
According to the comment chain in the JIRA ticket, "last used" is not just `state init`, but for any read/write operation using the State Tool. I don't know if that's covered in this PR, but this comment is not accurate.

Reply:
The value is being set as part of the `StoreProjectMapping` function which is called every time a project file is parsed. You can see in the integration test that I checkout a project first to set the last-used value.

Reply:
That's fair. Updated the comment to hopefully be more clear.